test/unittests/test_excel_to_json_lists.py

```diff
@@ -140,9 +144,9 @@ def test_excel2lists(self) -> None:
 
         # make sure that the invalid lists raise an Error
         with self.assertRaisesRegex(BaseError, r"Found duplicate in column 2, row 9"):
-            e2l.excel2lists(excelfolder="testdata/invalid-testdata/excel2json/lists-invalid-1", path_to_output_file=outfile)
+            e2l.excel2lists(excelfolder="testdata/invalid-testdata/excel2json/lists-invalid-1")
         with self.assertRaisesRegex(BaseError, r"The Excel file with the language code 'de' should have a value in row 10, column 2"):
-            e2l.excel2lists(excelfolder="testdata/invalid-testdata/excel2json/lists-invalid-2", path_to_output_file=outfile)
```

Comment:
`path_to_output_file=outfile` raised a pylint error. After examining, I realized that the parameter `path_to_output_file` is optional, and that its presence/absence doesn't influence the outcome of the test.